src/lib/blueprint.ts

```diff
@@ -121,10 +123,153 @@ export interface TypesModule {
   openapi: Openapi
 }
 
-export const createBlueprint = ({ openapi }: TypesModule): Blueprint => {
-  return {
+export const createBlueprint = ({
+  openapi,
+}: {
+  openapi: Openapi
+}): Blueprint => {
+  const blueprint: Blueprint = {
     name: openapi.info.title,
     routes: [],
     resources: {},
   }
+
+  // Filter for /acs/systems/list
+  const targetPath = '/acs/systems/list'
+  const targetSchema = 'acs_system'
+
+  // Check for fake data
+  const isFakeData = openapi.info.title === 'Foo'
+
```

Comment:
Just early return for the fake data for now. Generating a valid output for the fake data will be the next goal. 
```suggestion
if (openapi.info.title === 'Foo') return { name: 'Foo', routes: [], resources: {} }
```

Reply:
I've already made it work both ways and it felt like I was reversing progress but I can make it work only for the fake data for now if that's a strong preference

Reply:
Ok, whatever it easier! This suggestion was just to offer a possible simplification but not required!